docs/user-guide/admin-cli.md

```diff
@@ -317,22 +317,20 @@ Remove a tenant, optionally all volumes for a tenant can be removed as well.
 
 Sample:
 ```
-/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant rm tenant1 --remove-volumes True
+/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant rm --name=tenant1 --remove-volumes
```

Comment:
Do you mean 
```
vmdkops_admin tenant create T1
vmdkops_admin tenant rm T1
```
Or
```
vmdkops_admin tenant create --name=T1
vmdkops_admin tenant rm --name=T1
```
Most of tenant commands accept tenant name in the form of "--name=T1" for e.g. `vmdkops_admin tenant access ls --name=T1` So, I changed rm to accept  "--name=T1"